dist/configure

```diff
@@ -277,58 +284,101 @@ if [[ $target_arch == "aarch64" ]]; then
   echo "MARCH = aarch64" >> Makefile.config
 fi
 
+supports_128=false
+supports_256=false
+supports_vale=false
+supports_inline_asm=false
+supports_intrinsics=false
+
 if detect_arm; then
   echo "... detected ARM platform"
-  echo "... $build_target does not support 256-bit arithmetic"
-  echo "BLACKLIST += $(ls *CP256*.c *_256.c *_Vec256.c | xargs)" >> Makefile.config
-  echo "... $build_target does not support legacy vale stubs"
-  echo "BLACKLIST += evercrypt_vale_stubs.c" >> Makefile.config
-  echo "#define Lib_IntVector_Intrinsics_vec256 void *" >> config.h
+  # TODO: poorly worded; rename detect_arm_cc into detect_neon_compiler or
```

Comment:
Do you want to do this while you're at it? 🙂

Reply:
Done ;)